yb-voyager/cmd/importData.go

```diff
@@ -241,43 +241,69 @@ func discoverFilesToImport() []*ImportFileTask {
 
 func applyTableListFilter(importFileTasks []*ImportFileTask) []*ImportFileTask {
 	result := []*ImportFileTask{}
-	includeList := utils.CsvStringToSlice(tconf.TableList)
-	log.Infof("includeList: %v", includeList)
-	excludeList := utils.CsvStringToSlice(tconf.ExcludeTableList)
-	log.Infof("excludeList: %v", excludeList)
+
+	migInfo, err := LoadMigInfo(exportDir) //TODO: handle with msr.SourceDBConf
+	if err != nil {
+		utils.ErrExit("failed to load migration info: %s", err)
+	}
+	source.DBType = migInfo.SourceDBType
+	source.Schema = migInfo.SourceDBSchema
```

Comment:
Sanyam might have merged his msr related changes. So you can use `msr.SourceDBConf`.